<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordSafeSelectionWidget</class>
 <widget class="QWidget" name="PasswordSafeSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PasswordSafeSelectionWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="buttonFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="openButton">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="createButton">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="passwordSafeList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
